### DISCUSSION TITLE
Create Table & Insert Value

### DISCUSSION THREAD

**Discussion by ferrry1090** [cyrilgdn/terraform-provider-postgresql#20] [Q&A] [answered]
Hello Guys,

I am new to Terraform and I just found this Postgres provider to create Database, Schema and also config the User Role.
But in the documentation I could not find a way to create a table and insert value into the table.
Do you have any idea on this?

Thanks

**Comment by angaz** [comment, marked as answer]
Hi @ferry1090. What you are describing is best handled by database migration tools. I explain why in #17.
